packages/core/src/plugin/default-search-plugin/search-strategy/mysql-search-strategy.ts

```diff
@@ -40,8 +41,8 @@ export class MysqlSearchStrategy implements SearchStrategy {
         const facetValuesQb = this.connection
             .getRepository(ctx, SearchIndexItem)
             .createQueryBuilder('si')
-            .select(['MIN(productId)', 'MIN(productVariantId)'])
-            .addSelect('GROUP_CONCAT(facetValueIds)', 'facetValues');
+            .select(['MIN(si.productId)', 'MIN(productVariantId)'])
```

Comment:
It's missing the `si.` in front of the `productVariantId` here I believe (this caused an error in manual testing I did).

Reply:
Yeah, good catch, I don't have mysql environment for proper testing. I had a hope that e2e test will cover everything.